docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -111,7 +111,13 @@ innerBlocks: [
 ],
 ```
 
-The other would be to register patterns specific to your variation, we can cover that in another guide.
+By having `innerBlocks` in your variation you essentially skip the set up phase of Query Loop block with suggested patterns and the block is inserted with these inner blocks as its starting content.
+
+The other way would be to register patterns specific to your variation, which are going to be suggested in set up and replace flows of the block.
+
+Query Loop determines if there is an active variation of the block and if there are available specific patterns for this variation. If there are, these patterns are going to be the only ones suggested to the user, without including the default ones for Query Loop. If there are no such patterns, the default ones for Query Loop are going to be suggested.
```

Comment:
```suggestion
The Query Loop block determines if there is an active variation of itself and if there are specific patterns available for this variation. If there are, these patterns are going to be the only ones suggested to the user, without including the default ones for the original Query Loop block. Otherwise, if there are no such patterns, the default ones are going to be suggested.
```